co2eq/hotelpercountry.js

```diff
@@ -0,0 +1,42 @@
+import countries from './countries.json';
+
+export const modelName = 'hotelpercountry';
+export const modelVersion = '1';
+export const explanation = {
+  text: 'Calculations take into account the emissions of a stay depending of the country',
+  links: [
+    { label: 'UK Government Conversion Factor', href: 'https://docs.google.com/spreadsheets/d/1f1j9EeVn9czOZBJKLXvgPwnmldakxuJ7/edit#gid=1584958883' },
```

Comment:
let's link directly to the source (and not my file): https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2019

Reply:
Done :)